packages/tools/pluggable-widgets-tools/scripts/e2e.cypress.js

```diff
@@ -138,9 +138,9 @@ async function main() {
             "docker run -t " +
                 `-v ${REPO_ROOT}:/source ` +
                 `-v ${REPO_ROOT}/node_modules:/source/node_modules:ro ` +
-                "-w /e2e --name cypress cypress/included:9.6.1 " +
-                `--browser ${browserCypress} ${headedMode} --config baseUrl=http://${ip}:${freePort},video=true,videoUploadOnPasses=false,viewportWidth=1280,viewportHeight=1080,chromeWebSecurity=false ` +
-                `--config-file false --project ${process.cwd().replace(REPO_ROOT, "/source")}`,
+                "-w /e2e --name cypress cypress/included:10.4.0 " +
+                `--browser ${browserCypress} ${headedMode} --e2e --config-file /source/configs/e2e/cypress/cypress.config.js --config baseUrl=http://${ip}:${freePort},video=true,videoUploadOnPasses=false,viewportWidth=1280,viewportHeight=1080,chromeWebSecurity=false ` +
```

Comment:
Maybe put every cypress argument (which start with `--`) on their own line, this could increase readability of the arguments when they are in "column" representation.

Reply:
Thanks! Done! ✅